### PR TITLE
OBPIH-5414 See product name translation based on "product locale" (Scope: returns)

### PR DESCRIPTION
**Pull request by alannadolny**

### CODE REVIEW

File: src/js/components/returns/inbound/SendInboundReturn.jsx [hunk 1]
```diff
@@ -133,24 +133,23 @@ const FIELDS = {
...
         label: 'react.stockMovement.product.label',
         defaultMessage: 'Product',
         flexWidth: '2',
         headerAlign: 'left',
+        getDynamicAttr: ({ fieldValue }) => ({
+          tooltipValue: fieldValue?.name,
+        }),
         attributes: {
-          showValueTooltip: true,
+          formatValue: value => (
```
Comment (drodzewicz) [inline src/js/components/returns/inbound/SendInboundReturn.jsx:146]: good job on this refactor

**Review by drodzewicz — CHANGES_REQUESTED** [review]
you have pushed `lib > classpath.index` file which is probably not supposed to be in this PR.
Please remove it

other than that LGTM

File: grails-app/controllers/org/pih/warehouse/api/StockTransferApiController.groovy [hunk 2]
```diff
@@ -42,7 +42,6 @@ class StockTransferApiController {
             render([errorMessage: message] as JSON)
             return
         }
-
```
Comment (kchelstowski) [inline grails-app/controllers/org/pih/warehouse/api/StockTransferApiController.groovy:45]: is it necessary?

**Review by kchelstowski — CHANGES_REQUESTED** [review]

**Review by drodzewicz — APPROVED** [review]

**Review by jmiranda — COMMENTED** [review]
LGTM

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**